components/gc-page-success/gc-page-success-fr.html

```diff
@@ -0,0 +1,101 @@
+---
+{
+	"altLangPage": "gc-page-success-en.html",
+	"altLangPrefix": "gc-page-success_this-var-is-broken-in-modern-gcweb",
+	"breadcrumb": [ "this-var-is-broken-in-modern-gcweb" ],
```

Comment:
FWIW these were basically a reminder for myself to create a spinoff issue or PR about those variables no longer working in modern GCWeb. WET 4.0 and older incarnations of GCWeb use those variable names - but many were left untouched when GCWeb switched over to Jekyll.

This PR was affected because I originally started experimenting with widget updates before GCWeb moved over to Jekyll. At first, I couldn't figure out why on earth my language toggle and breadcrumb links were broken despite being properly-defined lol. Later on I updated to the new variable names and changed the old ones' values into todo notes :P.